packages/flutter/lib/src/widgets/binding.dart

```diff
@@ -493,7 +493,7 @@ mixin WidgetsBinding on BindingBase, ServicesBinding, SchedulerBinding, GestureB
           },
       );
 
-      WidgetInspectorService.instance!.initServiceExtensions(registerServiceExtension);
+      WidgetInspectorService.instance.initServiceExtensions(registerServiceExtension);
```

Comment:
Okay, this commit removes the `!` again. Ignore my previous comment on this line.